doc/minversions.md

```diff
@@ -18,14 +18,24 @@ myst:
 # Minimum Supported Versions
 
 PyGMT has adopted [SPEC 0](https://scientific-python.org/specs/spec-0000/) alongside the
-rest of the scientific Python ecosystem, and therefore:
+rest of the scientific Python ecosystem, and will therefore:
 
-- Support for Python versions be dropped 3 years after their initial release.
-- Support for core package dependencies (NumPy, pandas, Xarray) be dropped 2 years after
-  their initial release.
+- Drop support for Python versions 3 years after their initial release.
+- Drop support for core package dependencies (NumPy, pandas, Xarray) 2 years after their
+  initial release.
 
-Similarly, the PyGMT team has decided to discontinue support for GMT versions 3 years
-after their initial release.
+In addition to the above, the PyGMT team has also decided to:
+
+- Drop support for GMT versions 3 years after their initial release, while ensuring at
+  least two latest minor versions remain supported.
+- Maintain support for optional dependencies for at least 1 year after their initial
+  release. Users are encouraged to use the most up-to-date optional dependencies where
+  possible.
+
+:::{note}
+The policy is not strictly enforced and occasional breaches of the policy are allowed
+after discussions within the PyGMT team.
```

Comment:
I think we should be specific that breaches to the policy are only for **core** and **optional** packages following the SPEC 0 note (i.e. Python and GMT versions are to be dropped strictly after 3 years). How about wording it like this?

```suggestion
The SPEC 0 policy is enforced on a best-effort basis, and the PyGMT team may decide to
drop support for core (and optional) package dependencies earlier than recommended for
compatibility reasons.
```